server/db/index.js

```diff
@@ -1,6 +1,64 @@
 const db = require('./db')
+const Sequelize = require('sequelize')
+
+const {User, Task, Organization} = require('./models/index')
+
 
 // register models
 require('./models')
 
+const UserOrganization = db.define('user_organization', {
+  role: {
+    type: Sequelize.ENUM,
+    values: ['owner', 'co-owner', 'user', 'admin']
+  },
+
+  OrgId: {
+    type: Sequelize.INTEGER,
+    references: {
+      model: Organization,
+      key: 'id'
+    }
+
+  },
+
+
+  UserId: {
+    type: Sequelize.INTEGER,
+    references: {
+      model: User,
+      key: 'id'
+    }
+  }
+
+})
+
+User.belongsToMany(Organization, {through: UserOrganization})
+Organization.belongsToMany(User, {through: UserOrganization})
+
+
+const UserTask = db.define('user_task', {
+  TaskId: {
```

Comment:
if UserTask model doesn't have any fields unique to it then it's a composite key and we won't need to define task or user ids, they will be autogenerated by the association